blocks/carousel/carousel.js

```diff
@@ -0,0 +1,47 @@
+import { button, span } from '../../scripts/dom-helpers.js';
+import { decorateIcons, loadCSS } from '../../scripts/lib-franklin.js';
+import { decorateAPICards } from '../cards/cards.js';
+
+async function decorateAPICarousel(block) {
+  const apiEndpoint = block.querySelector('a').href;
+  block.innerHTML = '';
+  const apiResponse = await fetch(apiEndpoint);
+
+  if (!apiResponse.ok) {
+    return;
+  }
+
+  const apiInfo = JSON.parse(await apiResponse.text());
+  block.append(decorateAPICards(apiInfo.offers, true));
+}
+
+function navLeft(block) {
+  const ul = block.querySelector('ul');
+  ul.scrollLeft -= 600;
```

Comment:
I don't think having `600` hardcoded is a good approach. This number should be computed based on the card width and gap.